app/assets/stylesheets/custom.css.scss

```diff
@@ -895,6 +895,18 @@ $grayMediumLight: #eaeaea;
   /*float: right;*/
 }
 
+.rounded-box {
+  position: relative;
+  padding: 10px 10px 10px;
+  border: 1px solid #ddd;
+  -webkit-border-radius: 4px;
+  -moz-border-radius: 4px;
+  border-radius: 4px;
+  h4 {
+    color: #666;
```

Comment:
Color literals like `#666` should only be used in variable declarations; they should be referred to via variable everywhere else.